README.md

```diff
@@ -15,17 +16,52 @@ More information: [README](library/README.md)
 # Demo
 
 ## Live
+
 The current demo is continually updated and live for testing at: https://cybrid-app.github.io/cybrid-sdk-web/
 
 To login you will need to have the following:
+
 - Api keys (Id and Secret) or a valid Bearer token
 - Customer GUID
+- Some prior setup for your bank/customer (see setup below)
 
 Most browsers will block Cybrid api requests due to Cross-Origin Resource Sharing (CORS) rules. You must disable CORS to properly view the demo. Here's a handy script (for Mac OSX) that opens a developer friendly instance of Google Chrome:
+
 ```shell
 open -n -a /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --args --user-data-dir="/tmp/chrome_dev_test" --disable-web-security
 ```
 
+## API Setup
+
+The following setup should be preformed prior to using the Web Demo application.
+
+1. Create your customer using the `POST /api/customers` API
+2. Create a `type=fiat` account with `asset=USD` using `POST /api/accounts` API for your customer
+3. Create a `type=trading` account with `asset=BTC` using the `POST /api/accounts` (this is required to trade Bitcoin)
+4. Create a `type=trading` account with `asset=ETH` using the `POST /api/accounts` API (this is required to trade Ethereum)
+
+## Demo Setup
+
+The following are a set of first time steps you should follow in the Demo application.
+
+> NOTE: In sandbox no real identity information or banking credentials are required
+
+1. Preform an identity verification using the `identity-verification` component. Identity verifications can take some time because of the number of checks being preformed in the background. You may be asked to check back later for the status.
+
+If you call `GET /api/banks/{bank_guid}` you will get back your bank details, which includes the `features` your bank is configured with.
+
+If your bank is configured with a `backstopped_funding_source` the sandbox environment will automatically fund your fiat account for you. In production you will be required to make a book transfer via the transfers api.
+
+If your bank is configured with a `plaid_funding_source` you will need to add an external bank account.
+
+2. (`plaid_funding_source`) Connect a bank account using the `bank-account-connect` component. In sandbox mode you don't need to enter any real credentials into Plaid. Choose a bank from the list. Only select a single bank account (this is enforced in the production instance). If Plaid does ask for credentials:
+
+> Username: user_good \
+> Password: pass_good
+
+Once you have gone through Plaid and connected a bank account, the identity verification should be complete.\*\*
+\*\* 3. (`plaid_funding_source`) Transfer funds into the customer's fiat account using the `transfer` component. 4. Your customer can now preform trades using the `trade` component.
```

Comment:
Lines are messed up here. "3." and "4." should be on their own lines.

Moreover, we need to instruct people to only transfer $100 at a time.